### DISCUSSION TITLE
How it can i restore the configuration

### DISCUSSION THREAD

**Discussion by prabuanandan** [featurehub-io/featurehub#840] [General]
Hello, 
    How can i export the configurations like (users, api-key, feature flags etc) and use the same to create a new instance of it ?

Thanks,
Prabu

**Comment by rvowles** [comment]
Heya! You can export the database and then reimport it? Is that what you mean? You can also use the admin API if you wish to do more interesting things... Is this  what you are asking?

**Reply by prabuanandan** [reply]
Thanks a lot for replying . Yes you have answered my question. So the only way is to export the database is it? Is there a way i can import it as a json and reimport from the UI?

**Reply by rvowles** [reply]
Hiya! No, there hasn't been a request for such a feature, you are welcome to lodge an issue if you like?
